bin/chandra_repro

```diff
@@ -211,6 +213,10 @@ def update_aspsol_to_aspsolobi(asol_list,mydir, params):
     from ciao_contrib.runtool import make_tool
     from pycrates import read_file
 
+    if params["asol_update"] is False:
+        v1("Skipping boresite update to aspect solution file")
+        return asol_list
+
     v1("Applying boresite update to aspect solution file")
```

Comment:
If the asol update is applied, then gratings users must set `recreate_tg_mask=yes`.  The sky coordinates in the region block of the evt2 file are no longer valid.